@xen-orchestra/lite/src/components/pool/dashboard/cpuUsage/HostsCpuUsage.vue

```diff
@@ -13,9 +16,14 @@ import UsageBar from "@/components/UsageBar.vue";
 import type { Stat } from "@/composables/fetch-stats.composable";
 import { getAvgCpuUsage } from "@/libs/utils";
 import type { HostStats } from "@/libs/xapi-stats";
+import { useHostStore } from "@/stores/host.store";
 import { N_ITEMS } from "@/views/pool/PoolDashboardView.vue";
+import { storeToRefs } from "pinia";
 import { computed, type ComputedRef, inject } from "vue";
 
+const hostStore = useHostStore();
+const { hasError } = storeToRefs(hostStore);
```

Comment:
Please inline all others.

```suggestion
const { hasError } = storeToRefs(useHostStore());
```